client/src/index.js

```diff
@@ -16,18 +17,27 @@ const Routes = () => {
     <Container>
       <Router>
         <NavBar />
+        <Grid width={15} centered>
```

Comment:
You need to leave incoming changes in this file as your team mate changed the way the routes work in you project